tests/base.py

```diff
@@ -129,56 +144,46 @@ def expected_primary_keys(self):
 
     def expected_automatic_fields(self):
         """
-        return a dictionary with key of table name and value as the primary keys and replication keys
+        Return a dictionary with key of table name 
+        and value as the primary keys and replication keys
         """
         pks = self.expected_primary_keys()
         rks = self.expected_replication_keys()
 
         return {stream: rks.get(stream, set()) | pks.get(stream, set())
-                for stream in self.expected_streams()}
+                for stream in self.expected_streams(all_streams = True)}
 
     def expected_replication_method(self):
-        """return a dictionary with key of table name and value of replication method"""
+        """
+        Return a dictionary with key of table name 
+        and value of replication method
+        """
         return {table: properties.get(self.REPLICATION_METHOD, None)
                 for table, properties
                 in self.expected_metadata().items()}
 
-    def expected_streams(self):
-        """A set of expected stream names"""
-        return set(self.expected_metadata().keys())
+    def expected_streams(self, all_streams: bool = False):
```

Comment:
Instead of all_streams use only_trial_account_stream, take default value False

Reply:
updated